packages/web/src/scss/components/Drawer/_DrawerPanel.scss

```
@@ -0,0 +1,43 @@
+@use 'sass:map';
+@use 'theme';
+
+.DrawerPanel {
+    --drawer-translate-x: 0%;
+
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    width: theme.$drawer-panel-width;
```

```suggestion
    width: var(--#{tokens.$css-variable-prefix}drawer-panel-width, #{theme.$drawer-panel-width});
```

this would allow anyone to modify the width. We can do this either here or in theme.scss.